worktree_status.go

```diff
@@ -371,7 +371,7 @@ func (w *Worktree) doAdd(path string, ignorePattern []gitignore.Pattern, skipSta
 	}
 
 	if err != nil || !fi.IsDir() {
-		added, h, err = w.doAddFile(idx, s, path, ignorePattern)
+		added, h, err = w.doAddFile(idx, s, filepath.Clean(path), ignorePattern)
```

Comment:
The use of `path` in L376 can certainly do with a `filepath.Clean` as well. Let's do a `path = filepath.Clean(path)` in `L372` instead.

Reply:
OK, modifications done.